### ISSUE TITLE
`class_name` claims custom class hides global script class

### ISSUE THREAD

**Issue by lonegamedev** [godotengine/godot-vscode-plugin#650]
### Godot version

v4.2.1.stable.arch_linux

### VS Code version

1.88.1

### Godot Tools VS Code extension version

2.0.0

### System information

ArchLinux

### Issue description

This gets reported as hiding global class.

```
class_name GodotSucks
```

### Steps to reproduce

Create script class, 
assign class_name.

**Comment by DaelonSuzuka** [comment]
Yes, this is a known issue, it's been around forever. It's not caused by or influenced by this extension. It's a bug in the Language Server, which is part of the Godot Editor.

**Comment by lonegamedev** [comment]
Seems like problem is related to:
`https://github.com/godotengine/godot-vscode-plugin/issues/649#issuecomment-2066041382`

**Comment by HolonProduction** [comment]
Probably fixed by https://github.com/godotengine/godot/pull/104401

**Comment by Calinou** [comment]
Closing per the above comment. Please comment if you can still reproduce this on the latest Godot version *and* the latest version of this extension.
